pkgs/games/legendary-gl/default.nix

```diff
@@ -7,23 +7,27 @@
 
 buildPythonApplication rec {
   pname = "legendary-gl"; # Name in pypi
-  version = "0.20.1";
+  version = "0.20.6";
 
   src = fetchFromGitHub {
     owner = "derrod";
     repo = "legendary";
     rev = version;
-    sha256 = "0kdrhdvh9gxq1zinh5w852f5fjls9902pcrkhkhm2c0vvq7jfass";
+    sha256 = "1v6jbnasz2ilcafs6qyl6na4a8cxy2lgwr0hqsja6d846rfqa8ad";
   };
 
   propagatedBuildInputs = [ requests ];
 
   disabled = pythonOlder "3.8";
 
+  doCheck = false;
```

Comment:
```suggestion
  # no tests
  doCheck = false;
```

Reply:
I don't understand this. Obviously you have put a lot of thought into this, so there must be something I'm missing. In my mind, in every scenario I can think of it is better to have the checks turned off. Can you elaborate why we want them off?

Here's the scenarios I'm thinking of:

* The next version has no tests - Both `doCheck=true` and `false` are fine
* The next version has tests and they would succeed
  * `doCheck=true` performs the tests, which is good
  * `doCheck=false` does not perform the tests, which is ok but not best
* The next version has tests and they would fail because there is an actual bug
  * `doCheck=true` performs the tests and fails, forcing investigation which is good
  * `doCheck=false` does not perform the tests, buggy version is released, which is bad
* The next version has tests and they would fail because there is a problem with the way we perform the tests
  * `doCheck=true` performs the tests and fails, forcing investigation into why our infrastructure does not perform tests correcty, which is overall a good thing. In this instance, cutting a new issue to fix the infrastructure and switching to `doCheck=false` until it is fixed would be fine
  * `doCheck=false` does not perform the tests, problem in infrastructure is not discovered, which is ok but not ideal